pypdf/constants.py

```diff
@@ -451,6 +451,7 @@ class GraphicsStateParameters:
     SM = "/SM"
     SA = "/SA"
     BM = "/BM"
+    MASK = "/Mask"  # 1-bit image mask stream
```

Comment:
That does not belong here, but into the `ImageAttributes` class (PDF 1.7: "Table 89 – Additional Entries Specific to an Image Dictionary" or PDF 2.0: "Table 87 — Additional entries specific to an image dictionary")

Reply:
thanks : fixed